src/registry/inbuilt-components/inbuiltControls.ts

```diff
@@ -13,13 +13,26 @@ import OHRIDropdown from '../../components/inputs/select/ohri-dropdown.component
 import OHRITextArea from '../../components/inputs/text-area/ohri-text-area.component';
 import OHRIText from '../../components/inputs/text/ohri-text.component';
 import OHRIToggle from '../../components/inputs/toggle/ohri-toggle.component';
-import { UISelectExtended } from '../../components/inputs/ui-select-extended/ui-select-extended';
+import UISelectExtended from '../../components/inputs/ui-select-extended/ui-select-extended';
 import { OHRIRepeat } from '../../components/repeat/ohri-repeat.component';
 import { RegistryItem } from '../registry';
 
 /**
  * @internal
  */
+const controlTemplates = [
+  {
+    name: 'drug',
+    baseControlComponent: UISelectExtended,
+    datasource: {
+      name: 'drug_datasource',
+      config: {
+        class: '8d490dfc-c2cc-11de-8d13-0010c6dffd0f',
```

Comment:
@kajambiya is this a universal UUID for the drug class that can be found across all OpenMRS installations?

Reply:
Yes it is and even in Ampath the same approach is used. See [here](https://github.com/openmrs/openmrs-esm-form-entry/blob/c553a59f58f4e2aef3afd3fa3eaae0be4923c0df/src/app/form-data-source/form-data-source.service.ts#L144)